xml/cl.xml

```diff
@@ -5518,6 +5571,7 @@ server's OpenCL/api-docs repository.
         <extension name="cl_khr_icd" supported="opencl">
             <require>
                 <type name="CL/cl.h"/>
+                <type name="cl_icd_dispatch"/>
```

Comment:
I'm not sure what to do with this one.

It doesn't seem right to include it with `cl_khr_icd`, since it doesn't define any APIs that take a `cl_icd_dispatch` or a pointer to a `cl_icd_dispatch`, and the [extension text itself](https://registry.khronos.org/OpenCL/specs/3.0-unified/html/OpenCL_Ext.html#cl_khr_icd-inferring-vendors-from-function-call-arguments) only describes `struct _cl_icd_dispatch`.

Maybe it should be part of [cl_loader_layers](https://github.com/KhronosGroup/OpenCL-Docs/blob/main/ext/cl_loader_layers.asciidoc), since it at least defines a few APIs that take a pointer to a `cl_icd_dispatch`?

If we consider this type to be a part of `cl_loader_layers`, though, should we [typedef](https://github.com/KhronosGroup/OpenCL-Headers/blob/e049b16b5f157e2f28e7b5c301e71e1ccb3fe288/CL/cl_icd.h#L38) off of `struct _cl_icd_dispatch` though?  Based on the current headers the type doesn't appear to be associated with any extension...

Reply:
I suggest we move the require tag to cl_loader_layers, at least as a first step. It's obviously the case and results at least one user for this which is the goal of this change.